civis/resources/__init__.py

```diff
@@ -1,3 +1,3 @@
-from ._resources import generate_classes
+from ._resources import generate_classes, get_swagger_spec
 
-__all__ = ["generate_classes"]
+__all__ = ["generate_classes", "get_swagger_spec"]
```

Comment:
Yeah, I think you're right. I was debating the best way to do this since I think `get_swagger_spec` would be useful outside of `civis` while `generate_classes_maybe_cache` really isn't.